pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -459,6 +459,16 @@ func managePod(ctx context.Context, configMapsGetter corev1client.ConfigMapsGett
 		containerArgsWithLoglevel[0] += " --tls-cert-file=/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.crt"
 		containerArgsWithLoglevel[0] += " --tls-private-key-file=/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.key"
 	}
+
+	kubeControllerManagerConfigMap, err := configMapsGetter.ConfigMaps(required.Namespace).Get(ctx, "config", metav1.GetOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, false, err
+	}
+	var kubeControllerManagerConfig map[string]interface{}
+	if err := yaml.Unmarshal([]byte(kubeControllerManagerConfigMap.Data["config.yaml"]), &kubeControllerManagerConfig); err != nil {
```

Comment:
if it isn't found, `kubeControllerManagerConfigMap` could be nil

Reply:
since `Data` is `map[string]string` you can't have nil, at most it'll be an empty string. So that should work just fine.

Reply:
the map itself `if err != nil && !apierrors.IsNotFound(err) `

Reply:
IC, fixed.